customizations/mei-basic.xml

```diff
@@ -95,8 +95,9 @@
                 <!--<moduleRef key="MEI.msDesc"/>-->
                 <moduleRef key="MEI.midi" include="instrDef" />
                 <moduleRef key="MEI.namesdates" include="persName" />
+                <!-- the following elements are not available anymore in MEI.shared: arranger, composer, lyricist -->
                 <moduleRef key="MEI.shared" 
-                    include="accid arranger artic body caesura chord clef clefGrp composer date dir dynam ending label labelAbbr layer lb lyricist mdiv mei music note ornam pb pgFoot pgHead pubPlace rend respStmt rest sb score scoreDef section space staff staffDef staffGrp symbol syl tempo title "/>
+                    include="accid artic body caesura chord clef clefGrp date dir dynam ending label labelAbbr layer lb mdiv mei music note ornam pb pgFoot pgHead pubPlace rend respStmt rest sb score scoreDef section space staff staffDef staffGrp symbol syl tempo title "/>
```

Comment:
@lpugin Can you have a look if there are any side effects for MEI Basic (MuseScore Import etc.) when removing these elements from shared. Do we need to further adjust the customization?